docs/content/cluster-scope/quotas.md

```diff
@@ -9,6 +9,10 @@ When requesting a namespace, you can choose from 1 of 4 tier options for your re
 | Medium  |     2     |   8Gi  |   40Gi  |
 | Large   |     4     |  16Gi  |   80Gi  |
 
+To understand the difference between cpu limits/requests, please refer [this][k8s-doc]. 
```

Comment:
Change `please refer [this][k8s-doc]` to `please refer to [this][k8s-doc]`

Reply:
done